### PR TITLE
MTC docs update 2021

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected => This MTC doc update is against the `mtc-docs` branch.
- [na] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [na] All tests and CI builds passing

### Description

This PR updates the documentation following the addition of new features in #686, #737, #743, and #746.

### CODE REVIEW

File: docs/user/merging-feeds.md [hunk 1]
```diff
@@ -27,18 +32,31 @@ There are a set of rules that govern the requirements for input feed versions an
...
-  removed/ignored. `stop_time` records for the ignored trips shall also be removed.
-  If a `service_id` from the active calendar has only the `end_date` in the future, the `end_date`
-  shall be set to one day prior to the earliest `start_date` in future dataset before appending
-  the calendar record to the merged file.
-  `trip_ids` between active and future datasets must not match. If any `trip_id` is found to be
-  matching, the merge should fail with appropriate notification to user with the cause of the
-  failure. Notification should include all matched `trip_ids`.
+
+1. Trips will be merged as follows:
+   1. All `trip_id`s from one version and that is not in the other version will be added to the merged feed.
```
Comment (miles-grant-ibigroup) [inline docs/user/merging-feeds.md:37]: Should this be "and are not"?

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks great! Just a few potential typos to take a look at.

File: docs/user/merging-feeds.md [hunk 2]
```diff
@@ -27,18 +32,31 @@ There are a set of rules that govern the requirements for input feed versions an
...
+
+   2. Reusing `trip_id`s between the feed versions is permitted,
+      however the **trip signature** must all be identical in both feeds.
+      Two trips have the save signature if all their `arrival_time`, `departure_time`, `stop_id`, and `stop_sequence`
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply docs/user/merging-feeds.md]: Yep, fixed in https://github.com/ibi-group/datatools-ui/pull/751/commits/5c28282f6fe064a63c6703a0fa25110399794306.

File: docs/user/publishing-feeds.md [hunk 3]
```diff
@@ -1,16 +1,37 @@
...
+
+To avoid repeating the manual publishing steps above, 
+you can set up auto publishing for feeds that are to be fetched automatically.
+
+When creating new feeds, the auto-publishing setting is under the `Automatic publishing` section.
+For existing feeds, the auto-publishing setting is found under the `Settings` tab, `Auto-publish` category.
+
+![Auto-publish settings](../img/auto-publish-feed.png)
+
+In the list of feed sources, feeds that are set to be auto-published have an indicator displayed the feed name.
```
Comment (philip-cline) [inline docs/user/publishing-feeds.md:21]: Think the word "below" might have been missed.
```
In the list of feed sources, feeds that are set to be auto-published have an indicator displayed below the feed name.
```

**Review by philip-cline — APPROVED** [review]
One tiny typo but otherwise looking good to me

File: docs/user/publishing-feeds.md (hunk 3 above)
Comment (binh-dam-ibigroup) [reply docs/user/publishing-feeds.md]: Good catch! Thank you, fixed in https://github.com/ibi-group/datatools-ui/pull/751/commits/add3f0bc629feb9e14309b728be9311266e56f53.

**Pull request merged**